src/mm-input/mm-input.js

```diff
@@ -34,6 +34,11 @@
 				value: false,
 				reflectToAttribute: true
 			},
+			readonly: {
```

Comment:
my thinking for this one is because the attribute on native `<input>` is just `readonly`, does it make more sense for us to use camel?